src/main/java/com/iexec/worker/executor/TaskManagerService.java

```diff
@@ -233,6 +268,11 @@ ReplicateActionResponse compute(String chainTaskId) {
                     context, chainTaskId);
         }
 
+        if (taskDescription.isTeeTask() && !lasServicesManager.startLasService(chainTaskId)) {
```

Comment:
1. This is only true for a Scone task
2. I suggest to move that specific part within the `TeeService`
What do you think?

Reply:
Good idea, please see https://github.com/iExecBlockchainComputing/iexec-worker/pull/464/commits/75e674171d17f09f8e67bebd6489f7d7bf81b06e.